<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1402</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1381</width>
      <height>751</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1381</width>
      <height>751</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1381</width>
      <height>10000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QLabel" name="temp_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Temperature [C]</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="hum_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Humidity [% RH]</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="temp_lcdNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="hum_lcdNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="ReadMeasurementsButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sensor Read
Mesurements</string>
      </property>
     </widget>
     <widget class="QLabel" name="hum_label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pressure [hPa]</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="press_lcdNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>1171</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Temperature</string>
       </attribute>
       <widget class="PlotWidget" name="tempGraphWidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>961</width>
          <height>330</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="tempPlotLen">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
       <widget class="QLabel" name="baud_label_4">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>10</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Visible measurements</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Humidity</string>
       </attribute>
       <widget class="PlotWidget" name="humGraphWidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>961</width>
          <height>330</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="humPlotLen">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
       <widget class="QLabel" name="baud_label_5">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>10</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Visible measurements</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Pressure</string>
       </attribute>
       <widget class="PlotWidget" name="pressGraphWidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>961</width>
          <height>330</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="baud_label_6">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>10</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Visible measurements</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="pressPlotLen">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Config</string>
     </attribute>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>460</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Send Command</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="send_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="InitButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sensor
Initialize</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeinitButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sensor
Deinitialize</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="globalResponse">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>1351</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>430</y>
        <width>1351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>460</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Clear Display</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>-10</y>
        <width>21</width>
        <height>451</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="temp_label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Temperature Measurement</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="temp_meas_sel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Disabled</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x16</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="temp_label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Humidity Measurement</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="hum_meas_sel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Disabled</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x16</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="temp_label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pressure Measurement</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="press_meas_sel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Disabled</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oversampling x16</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="temp_label_5">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sensor Wroking Mode</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="mode_sel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Forced</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Auto</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="temp_label_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Standby Time</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="s_time_sel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0.5 ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10 ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20 ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>62.5 ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>125 ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>250 ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1000 ms</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="temp_label_7">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>150</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>IIR filter</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="filter_sel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>170</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Off</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 coefficients</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 coefficients</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8 coefficients</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16 coefficients</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="portSelectBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="portRefreshButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="baud_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Baudrate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="baudrateLineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string>115200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="port_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1402</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
